src/components/select/Select.js

```diff
@@ -42,26 +42,26 @@ const Select = ({
     showLabel = true,
     ...rest
 }: Props) => {
-    const classes = classNames(className, 'select-input-container', {
-        'show-error': !!error || showErrorOutline,
-        'is-disabled': isDisabled,
+    const classes = classNames(className, 'bdl-Select', {
+        'bdl-show-error': !!error || showErrorOutline,
+        'bdl-is-disabled': isDisabled,
     });
 
     return (
         <div className={classes}>
             <Label hideLabel={!showLabel} text={label} tooltip={labelTooltip}>
                 <Tooltip isShown={!!error} position="middle-right" text={error || ''} theme="error">
-                    <span className="select-container">
-                        <span className="select-container-inner">
+                    <span className="bdl-Select-container">
```

Comment:
Can we avoid "container" by using something like `bdl-Select-main/bdl-Select-mainContent` or `bdl-Select-content/bdl-Select-contentBody`?